src/workspace-hack/Cargo.toml

```diff
@@ -116,6 +116,8 @@ sqlx-postgres = { version = "0.7", default-features = false, features = ["any",
 sqlx-sqlite = { version = "0.7", default-features = false, features = ["any", "chrono", "json", "migrate", "offline", "time", "uuid"] }
 strum = { version = "0.25", features = ["derive"] }
 subtle = { version = "2" }
+tikv-jemalloc-sys = { git = "https://github.com/risingwavelabs/jemallocator.git", rev = "64a2d9", features = ["profiling", "stats", "unprefixed_malloc_on_supported_platforms"] }
+tikv-jemallocator = { git = "https://github.com/risingwavelabs/jemallocator.git", rev = "64a2d9", features = ["profiling", "stats", "unprefixed_malloc_on_supported_platforms"] }
```

Comment:
I feel this is ok. Test docker pipeline just in case. https://buildkite.com/risingwavelabs/docker/builds/14401

Reply:
Failed 🤣 Let me check...

Reply:
I struggled to understand the exact reason (since it's not in the `build-dependencies` now), but the simplest fix is to exclude `tikv-jemalloc-sys` in `hakari.toml`